### PR TITLE
Refactored tests for jax.random such that they work with all backends

### PR DESCRIPTION
**Pull request by fspyridakos**
Modified the tests to call the PRNG function directly because otherwise we would need to generate a `uint32` PRNGKey. Now the tests can pass for all backends (even if some fail now for backend related issues)

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_random.py [hunk 1]
```diff
@@ -1,203 +1,104 @@
...
-        min_value=0,
-        max_value=2000,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
-        max_dim_size=2,
-    ),
+    minval, maxval = minval_maxval
+
+    jax_frontend.config.update("jax_enable_x64", True)
```
Comment (CatB1t) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_random.py:38]: Wouldn't that still be enabled after the test finishes? maybe we should refactor the x64 enable code into a context manager and use it in JAX random tests so that after the test finishes it will clean up and reset the flag to False.

**Review by CatB1t — CHANGES_REQUESTED** [review]
Great work @fspyridakos! just a small point mentioned, happy to hear your thoughts.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
